integration/tests/conftest.py

```diff
@@ -835,3 +881,39 @@ def alt_contract(accounts, web3_client):
 @pytest.fixture(scope="session")
 def default_cu_price(pytestconfig: Config) -> int | None:
     return pytestconfig.environment.default_cu_price  # must be equal to compose.proxy.environment.DEFAULT_CU_PRICE
+
+
+@pytest.fixture(scope="session")
+def neon_api_client(environment: EnvironmentConfig) -> NeonApiClient:
```

Comment:
it's better to move back to neon_evm conftest because for proxy tests we don't have neon_api_client in env